Flutter/linear-gauge/range.md

```diff
@@ -71,6 +71,44 @@ However, to draw concave, convex and exponential-like shapes the [`midValue`](ht
 
 ![Draw linear gauge concave range](images/gauge-range/range-concave.png)
 
+## Customize edge style
+
+The [`edgeStyle`](https://pub.dev/documentation/syncfusion_flutter_gauges/latest/gauges/LinearGaugeRange/edgeStyle.html) property customizes how the edges of a range should be appear in linear gauge. The available options are `startCurve`, `endCurve`, `bothCurve`, and `bothFlat` options. 
```

Comment:
The [`edgeStyle`] property defines the appearance of either sharp or curved edges for a LinearGaugeRange.